src/decorators/bootstrap/array.html

```diff
@@ -0,0 +1,32 @@
+<div  class="schema-form-array {{form.htmlClass}}"
```

Comment:
also how much did you have to change these files? it would be nice if we could keep track of how much we were having to change the OOTB decorators we copied these from.  Also should probably have a comment saying which version of the decorators we based it off of in case we need / want to update them.